install.sh

```diff
@@ -306,7 +306,7 @@ nvm_do_install() {
     command printf "${SOURCE_STR}"
   else
     case "${NVM_PROFILE-}" in
-      ".bashrc" | ".bash_profile" | ".zshrc")
+      *"/.bashrc" | *"/.bash_profile" | *"/.zshrc")
```

Comment:
Since I can't run the tests locally, I might mess it up when manually mocking up the variables, or something like that, I'll be happy to write test, which part would you like to have?